java/dataset/src/test/java/org/apache/arrow/dataset/file/TestFileSystemDataset.java

```diff
@@ -357,6 +364,41 @@ public void testBaseArrowIpcRead() throws Exception {
     AutoCloseables.close(factory);
   }
 
+  @Test
+  public void testBaseOrcRead() throws Exception {
+    String dataName = "test-orc";
+    String basePath = TMP.getRoot().getAbsolutePath();
+
+    TypeDescription orcSchema = TypeDescription.fromString("struct<ints:int>");
+    Writer writer = OrcFile.createWriter(new Path(basePath, dataName),
+            OrcFile.writerOptions(new Configuration()).setSchema(orcSchema));
+    VectorizedRowBatch batch = orcSchema.createRowBatch();
+    LongColumnVector longColumnVector = (LongColumnVector) batch.cols[0];
+    longColumnVector.vector[0] = Integer.MIN_VALUE;
+    longColumnVector.vector[1] = Integer.MAX_VALUE;
+    batch.size = 2;
+    writer.addRowBatch(batch);
+    writer.close();
```

Comment:
Could be possible to externalize this as a common method? something like this OrcWriteSupport.writeTempFile

Reply:
Thanks @davisusanibar, sure! Extract it into OrcWriteSupport.writeTempFile

Reply:
Thank you